images/oauth2-proxy/configs/latest.apko.yaml

```diff
@@ -19,7 +19,7 @@ accounts:
   run-as: oauth2-proxy
 
 entrypoint:
-  command: /bin/oauth2-proxy
+  command: /usr/bin/oauth2-proxy
```

Comment:
@found-it - any issue with this?

Reply:
Looks like this was my change when I moved this to the standard go/build pipeline: https://github.com/wolfi-dev/os/blob/8259eca08a08507733990a2c94e0b3d6d6425d8d/oauth2-proxy.yaml#L32

/usr/bin is our standard here for these, fwiw

Reply:
Yep as long as `/usr/bin` is the standard (just separately confirmed it's where the oauth2-proxy binary should be) then there is no issue @jdolitsky